src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/sdl_pending_resumption_handler.cc

```diff
@@ -164,7 +166,7 @@ void SDLPendingResumptionHandler::on_event(
     }
     freezed_resumptions_.clear();
   } else {
-    LOG4CXX_DEBUG(logger_, "Resumption of subscriptions is NOT successful");
+    LOG4CXX_DEBUG(logger_, "Ressubsumption of subscriptions is NOT successful");
```

Comment:
@LuxoftAKutsan please revert this change

Reply:
Fixed